src/ScottPlot5/ScottPlot/DataSource/XYArrays.cs

```diff
@@ -4,7 +4,7 @@
 /// This data source has Xs and Ys defined in fixed-length arrays.
 /// Changes made to the contents of the arrays will appear when the plot is rendered.
 /// </summary>
-public class XYArrays : IDataSource
+public class XYArrays : ICoordinateSource
```

Comment:
Weird name, perhaps `CoordinateArray` would be better?

Reply:
For the interfaces I went with `IScatterSource` and `ISignalSource`

Example scatter types are
* `ScatterCoordinateArray`
* `ScatterCoordinateList`
* `ScatterXYArrays`
* `ScatterXYLists`